src/main/resources/alfresco/transforms/0100-basePipelines.json

```diff
@@ -210,6 +252,17 @@
         "pdfRendererOptions",
         "imageMagickOptions"
       ]
+    },
+    {
+      "transformerName": "outlookMsgToLibreofficeViaText",
+      "transformerPipeline" : [
+        {"transformerName": "OutlookMsg",                              "targetMediaType": "text/plain"},
+        {"transformerName": "libreoffice"}
+      ],
+      "supportedSourceAndTargetList": [
+      ],
+      "transformOptions": [
+      ]
```

Comment:
Should these have options? "tikaOptions" are valid for the "OutlookMsg" transformer